mission/mission.proto

```diff
@@ -6,25 +6,63 @@ option java_package = "io.dronecore.mission";
 option java_outer_classname = "MissionProto";
 
 service MissionService {
-    rpc SendMission(SendMissionRequest) returns(MissionResult) {}
-    rpc StartMission(StartMissionRequest) returns(MissionResult) {}
+    rpc UploadMission(UploadMissionRequest) returns(UploadMissionResponse) {}
+    rpc DownloadMission(DownloadMissionRequest) returns(DownloadMissionResponse) {}
+    rpc StartMission(StartMissionRequest) returns(StartMissionResponse) {}
+    rpc PauseMission(PauseMissionRequest) returns(PauseMissionResponse) {}
+    rpc SetCurrentMissionItemId(SetCurrentMissionItemIdRequest) returns(SetCurrentMissionItemIdResponse) {}
+    rpc GetCurrentMissionItemId(GetCurrentMissionItemIdRequest) returns(GetCurrentMissionItemIdResponse) {}
+    rpc GetMissionSize(GetMissionSizeRequest) returns(GetMissionSizeResponse) {}
```

Comment:
```
message GetMissionCountRequest {}
message GetMissionCountResponse {
    int32 mission_count = 1;
}
rpc GetMissionCount(GetMissionCountRequest) returns(GetMissionCountResponse) {}
```
Isn't this intuitive ?

Reply:
Not sure about this one. But it's right that `MissionSize` might sound like "the distance covered by the mission" or something like this.

But I don't know if `MissionCount` is intuitive to me (but if it is for the others, I'm fine with it).

Or maybe `MissionItemsCount` or `MissionItemsNumber`?

I'd like to get more opinions on this one :-).

Reply:
Same as that of [MISSION_COUNT](http://mavlink.org/messages/common#MISSION_COUNT) in MAVLink.

Reply:
`MissionItemCount`  is also fine I think.

Reply:
If that's the MAVLink message, then I guess it makes sense to use it!